python_modules/dagster/dagster_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -1997,6 +1998,98 @@ def _fetch_counts(storage, after_cursor=None):
                     )
                     assert _fetch_counts(storage, after_cursor=9999999999) == {c: {}, d: {}}
 
+    def test_get_latest_asset_partition_materialization_attempts_without_materializations(
+        self, storage, instance
+    ):
+        a = AssetKey(["a"])
```

Comment:
nit: can we add another test for that adds events for another asset key? so that we can confirm that events from other assets aren't returned